src/matched_data.ts

```diff
@@ -52,3 +69,15 @@ function decodebin(enc: ArrayBuffer) {
       };
   }
 }
+
+function b64decode(base64: string) {
```

Comment:
I think this is the same as `Buffer.from(base64, 'base64').buffer`, if you're fine using the nodejs compat flag